docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+# Copy this into a file called `docker-compose.yml`
+version: "3.2"
+
+    services:
+      todo_app_prod:
+        container_name: production
+        image: todo-app:prod
+        build: 
+          context: .
+          dockerfile: Dockerfile
+        volumes: 
+          - type: volume
+            source: ./todo_app
+            target: /todo_app/todo_app
+        ports:
+            - 5000:5000
+        env_file:
+            - .env
+
+      todo_app_dev:
+        container_name: Development
+        image: todo-app:dev
+        build: 
+          context: .
+          dockerfile: Dockerfile
+        volumes: 
+          - type: volume
+            source: ./todo_app
+            target: /todo_app/todo_app
+        ports:
+          - 5001:5000
```

Comment:
I suggested host port 5001 so that you could start up all the services in the docker-compose.yml file simultaneously with `docker compose up`.

If you only expect people to start one at a time, you could let the ports conflict and just tell people to make sure to specify a service with `docker compose up your_service-name`, or you could create different yml files to make that clearer and then you use different files with `docker compose up -f your_file_name.yml`